setup.cfg

```diff
@@ -101,57 +101,32 @@ install_requires =
     cryptography>=0.9.3
     deprecated>=1.2.13
     dill>=0.2.2
-    # Flask and all related libraries are limited to below 2.0.0 because we expect it to introduce
-    # Serious breaking changes. Flask 2.0 has been introduced in May 2021 and 2.0.2 version is available
-    # now (Feb 2022): TODO: we should attempt to migrate to Flask 2 and all below flask libraries soon.
-    flask>=1.1.0, <2.0
-    # We are tightly coupled with FAB version because we vendored in part of FAB code related to security manager
-    # This is done as part of preparation to removing FAB as dependency, but we are not ready for it yet
-    # Every time we update FAB version here, please make sure that you review the classes and models in
-    # `airflow/www/fab_security` with their upstream counterparts. In particular, make sure any breaking changes,
-    # for example any new methods, are accounted for.
```

Comment:
I think we should keep this block to explain the `==` pin.

Reply:
Yep. To much of a cleanup :)